src/protocols/storage_manager.js

```diff
@@ -0,0 +1,84 @@
+const Packet = require('../packet'); // eslint-disable-line no-unused-vars
+const PacketV0 = require('../packetv0'); // eslint-disable-line no-unused-vars
+const PacketV1 = require('../packetv1'); // eslint-disable-line no-unused-vars
+const RMCMessage = require('../rmc'); // eslint-disable-line no-unused-vars
+const Stream = require('../stream');
+
+const Requests = require('./requests/storage_manager');
+const Responses = require('./responses/storage_manager');
+
+class StorageManager {
```

Comment:
A note about this being the old Utility protocol should be added for clarity purposes